scripts/conveyorBelt.js

```diff
@@ -0,0 +1,16 @@
+class ConveyorBelt extends Cube {
+  constructor(x, y, name, direction) {
+    super(x, y, AssetStore.getTexture(`conveyorBelt${direction}`));
```

Comment:
This is a reasonable solution, but it is actually possible to solve this with only canvas methods. The benefit would be only needing one texture. The con would be some extra code.

You see, you cant interact with an image after you have placed it on the canvas, therefore rotation becomes really complicated.

You have to:

1. Rotate the whole shebang around the position where the texture will be drawn.
2. Draw the texture.
3. Rotate the whole shebang to its original position.

You can see an example of this in my game here: https://github.com/Istanful/onyx/blob/d499b6951d9341ff98c75da32d999a6cb4bc02dd/scripts/GameObject.js#L252-L264

Reply:
Sounds complicated, but hey, I'll check it out B)